src/main/resources/CU-ApplicationResources.properties

```diff
@@ -671,6 +671,7 @@ create.accounting.document.principle.id.bad=Could not find person by principle i
 create.accounting.document.vendor.id.bad=Could not find Vendor by vendor id {0}.
 create.accounting.document.payee.type.code.bad=Invalid payee type code: {0}.
 create.accounting.document.payee.name.not.same.as.vendor=The payee name in KFS is {0} and the payee name entered in the xml is {1}.  The payee name from KFS will be used.  Please use the correct name.
+create.accounting.document.payee.address.id.with.non.vendor=Payee type code {0} and payee address id {1} were supplied.  THe payee address id may only be used with payee type of V.
```

Comment:
Minor typo in message: `THe payee` --> `The payee`

Also, should this message use a placeholder for the expected payee type value?

Reply:
fixed typo.  The error happens when the payee address ID is entered and it is not a vendor type payee.so I don't think we need a placeholder for that.

Reply:
As long as we don't anticipate changing the expected vendor type code, then I'm okay with leaving the message's placeholder setup as-is.